.circleci/config.yml

```diff
@@ -29,6 +29,33 @@ jobs:
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
               echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
               docker push valhalla/valhalla:build-latest
+            elif [[ ! -z "${CIRCLE_TAG}" ]]; then
+              echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+              docker push valhalla/valhalla:build-${CIRCLE_TAG}
+            fi
+
+  build-binary:
+    requires: build-base
```

Comment:
i want this one to depend on the build job but it seems circle doesnt like this syntax (maybe its from an older version)